Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean

```diff
@@ -18,10 +18,7 @@ We also prove basic properties of these functions.
 
 noncomputable section
 
-open scoped Classical
-open Real NNReal ENNReal ComplexConjugate
```

Comment:
I suspect a lot of these are actually mistranslations of `open_locale`; but  I guess that ship has long since sailed, and replacing the `open`s with `open scoped` that corresponded to `open_locale` in Lean 3 is far too much work now.